doc/fluid/design/dist_train/async_update.md

```diff
@@ -0,0 +1,52 @@
+# Design Doc: Asynchronous Update With Distributed Training
+
+## Background
+
+For the typical synchronous distributed training, some significant steps are as follows:
+
+1. A Trainer will compute the gradients and SEND them to the Parameter
+Server(PServer) nodes.
+1. After the PServer node received gradients came from all the Trainers, it would apply the gradient to the respective variables, and using an optimize algorithms(SGD,
```

Comment:
It will aggregate the gradient for the same parameter into one gradient and then apply the aggregated gradient to the parameter.

Reply:
Done.